src/pages/LocalAuthorityDashboard/ManageSchools/ManageSchools.module.scss

```diff
@@ -4,36 +4,39 @@
   @include g.page-container;
   padding: 48px 50px;
 
-  h1 {
-    color: g.$dark-blue;
-    padding: 20px 60px;
-  }
-}
+  .header {
```

Comment:
I've seen these styles repeated - worth doing a header mixin in the globals do you reckon?